central/plugins/org.jboss.tools.central/src/org/jboss/tools/central/editors/JBossCentralEditor.java

```diff
@@ -568,10 +570,14 @@ public void widgetDisposed(DisposeEvent arg0) {
 					}
 				});
 				// Early access installed
-				Job checkEarlyAccessJob = new UIJob("Check installation for Early Access") {
+				Job checkEarlyAccessJob = new Job("Check installation for Early Access") {
 					@Override
-					public IStatus runInUIThread(IProgressMonitor monitor) {
-						HeaderText.this.hasEarlyAccessIUs = InstallationChecker.getInstance().hasEarlyAccess();
+					public IStatus run(IProgressMonitor monitor) {
+						try {
+							HeaderText.this.installChecker = InstallationChecker.getInstance();	
```

Comment:
You're right. I don't get how I could miss this issue...
See amended commit. However, I prefer running as few things as possible in UIThread since it creates UI freezes and slowness. I prefered encapsulating the UI changes in a Display.exec().